plugins/linux-capture/pipewire.c

```diff
@@ -552,6 +552,7 @@ static void on_param_changed_cb(void *user_data, uint32_t id,
 	const struct spa_pod *params[3];
```

Comment:
Another tiny nit: please reword the title to `linux-capture: Fallback to SHM on older PipeWire versions`